tests/general/LineContinuationBearTest.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from queue import Queue
+
+from bears.general.LineContinuationBear import LineContinuationBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+good_file = """
+a = some_function(
+    '1' + '2')
+"""
+
+bad_file = r"""
+a = '1' \
+    '2'
+"""
+
+
+class LineContinuationBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = LineContinuationBear(self.section, Queue())
+
+    def test_run(self):
+        self.section.append(Setting('language', 'Python'))
+        self.check_results(
+            self.uut, bad_file.splitlines(),
+            [Result.from_values('LineContinuationBear',
+                                'Explicit line continuation is not allowed.',
+                                line=2, column=9, end_line=2, end_column=10,
```

Comment:
shouldn't this point to line 1 ? As the token for line continuation is on Line 1 and if only one line is shown in diff i.e.
```
    '2'
```
This would certainly be confusing for the user

Reply:
or the starting should be from line 1 and the end_line could be the next line

Reply:
Umm, no since you see my triple `"` start from line `1` and the line continuation delimiter is on line `2`.

Reply:
I initially thought of starting the code from line 1 itself but it was looking weird :sweat_smile: 

But I can move it to line 1, no problems there :)

Reply:
Why not put test files into a folder?